#!/usr/bin/env python
PACKAGE = "dynamic_parameter"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("T2S1gripper",    str_t,    0, "A string parameter",  "open")
gen.add("T2S2gripper",    str_t,    0, "A string parameter",  "close")
gen.add("T2S3gripper",    str_t,    0, "A string parameter",  "none")
gen.add("T3S1gripper",    str_t,    0, "A string parameter",  "none")
gen.add("T3S2gripper",    str_t,    0, "A string parameter",  "open")
gen.add("T3S3gripper",    str_t,    0, "A string parameter",  "none")

 

exit(gen.generate(PACKAGE, "dynamic_parameter", "PickPlaceGrip"))